conf/mini_provisioner/v2/setup.yaml

```diff
@@ -37,6 +37,14 @@ ha:
     upgrade:
         cmd: /opt/seagate/cortx/ha/bin/ha_setup upgrade
         args: --config $URL
+        pre:
+            cmd: /opt/seagate/cortx/ha/bin/ha_setup pre_upgrade
+            args: --config $URL

```

Comment:
currently the same URL as for deployment is passed here but  nothing special is added in scope of the upgrade

Reply:
I kept it similar to other commands without specific reason. This won't affect mini-provisioner behavior I think.